src/components/Banner/Banner.jsx

```diff
@@ -0,0 +1,152 @@
+// Node modules.
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+// Relative imports.
+import AlertBox from '../AlertBox/AlertBox';
+
+const DISMISSED_BANNERS_KEY = 'DISMISSED_BANNERS';
+
+export class Banner extends Component {
```

Comment:
Can you add some documentation to this component about how it manages state in `storage`? Specifically around the ability to dismiss multiple alerts. This change is what took me the longest to figure out.

Reply:
Sure! Where would be the best place for me to add the documentation, perhaps in the `Banner.mdx`? 🙂

Reply:
In a JSDoc right here. Storybook will pick it up. :grin:

:thinking: That `.mdx` file is I think no longer used. It used to be in the old documentation site, but now that Storybook is a thing, I don't think it's used.

Reply:
I updated the comment for `storage`, let me know if that's sufficient or if you were thinking of something else! 🙂 https://github.com/department-of-veterans-affairs/component-library/pull/147/commits/a000fdb9f980db38d8a30c5bd130c37fb70fc2ea

Reply:
![image](https://user-images.githubusercontent.com/12773166/131533984-f73db443-9f2b-4114-8742-52153ab8b93d.png)

It looks like all the components on `master` still have the `.mdx` files btw, lmk if you want me to delete them and I'm happy to include that in this PR 👍

Reply:
Nah, we'll catch that in another PR since it's not part of the changes here. Thanks, though!